looks good to me, though build fail looks real
```
======================================================================
ERROR: test_group_by_missing_from_columns (corehq.apps.userreports.tests.test_report_config.ReportConfigurationTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/userreports/tests/test_report_config.py", line 51, in test_group_by_missing_from_columns
    data_source = ReportFactory.from_spec(report_config)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/userreports/reports/factory.py", line 99, in from_spec
    from corehq.apps.userreports.reports.data_source import ConfigurableReportDataSource
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/userreports/reports/data_source.py", line 7, in <module>
    from apps.userreports.reports.specs import DESCENDING
ImportError: No module named apps.userreports.reports.specs
----------------------------------------------------------------------
```

--- a/corehq/apps/userreports/reports/data_source.py
+++ b/corehq/apps/userreports/reports/data_source.py
@@ -4,10 +4,10 @@ from sqlagg import (
     TableNotFoundException,
 )
 from sqlalchemy.exc import ProgrammingError
-from apps.userreports.reports.specs import DESCENDING
 from corehq.apps.reports.sqlreport import SqlData
 from corehq.apps.userreports.exceptions import UserReportsError
 from corehq.apps.userreports.models import DataSourceConfiguration
+from corehq.apps.userreports.reports.specs import DESCENDING
 from corehq.apps.userreports.sql import get_table_name
 from corehq.apps.userreports.views import get_datasource_config_or_404
 from dimagi.utils.decorators.memoized import memoized

